docs/antrea-agent-simulator.md

```diff
@@ -0,0 +1,53 @@
+# Run Antrea agent simulator
+
+This document describes how to run the Antrea agent simulator. The simulator is
+useful for Antrea scalability testing, without having to create a very large
+cluster.
+
+## Build the images
+
+  ```bash
+  make build-scale-simulator
+  ```
```

Comment:
nit: I don't think any of these codeblocks need to be indented

Reply:
@antoninbas thanks for the latest advices, just resolved